### ISSUE TITLE
The desktop client keeps running after a new version was installed

### ISSUE THREAD

**Issue by jnweiger** [owncloud/client#7706]
Seen on Linux, (may be other platforms too):

Desktop client 2.6.0 is up and running. 
`apt-get update; apt-get install owncloud-client` replaces the 2.6.0 files on disk with the 2.6.1 files.

The desktop client is still running and shows version 2.6.0
Calling `owncloudcmd --version` shows 2.6.1

Enhancement idea: 
Could the client monitor its binary file or some of its libraries on disk (size + date should be sufficient), and suggest a restart, when it finds that the files change?

**Comment by michaelstingl** [comment]
> Could the client monitor its binary file or some of its libraries on disk (size + date should be sufficient), and suggest a restart, when it finds that the files change?

@dschmidt reported, VS Code does something similar.

**Comment by tboerger** [comment]
This sounds like a totally common behavior to me. Some distros are sending emails in that case which shouldn't be the case for desktop systems.

**Comment by jnweiger** [comment]
Yes, as a Linux devops you learn that behaviour as normal. It still leaves room for improvement from an end user UX perspective.

Generating an e-mail in that situation seems out of context to me. Don't do it.

**Comment by TheOneRing** [comment]
@dragotin I think we have a unit test for this scenario, shouldn't it notify the user too?

**Comment by TheOneRing** [comment]
@jnweiger the client should already monitor the binary.
If it still doesn't work we need to fix it.
Can you reproduce the issue?
https://github.com/owncloud/client/blob/db125905e521afe61e5bd491fffad5191d4007b8/src/gui/updater/ocupdater.cpp#L540-L552

**Comment by TheOneRing** [comment]
There was an issue with the --version parser so the detection was broken.
Can someone retry the scenario?
